vm/entrypoint/src/syscalls/mod.rs

```diff
@@ -30,3 +30,4 @@ pub const HOST_READ: u32 = 0x00_00_00_A0;
 pub const HOST_WRITE: u32 = 0x00_00_00_A1;
 pub const HOST_CALL: u32 = 0x00_00_00_A2;
 pub const HOST_GETBALANCE: u32 = 0x00_00_00_A3;
+pub const HOST_SPAWN: u32 = 0x00_00_00_A4;
```

Comment:
Because the code in `vm/entrypoint` is _guest_ code. The guest cannot rely on `core`. There's some duplication as a result but I think it's okay. I suppose we could set up a very simple 'types' crate, something that like, that both host and guest rely on.